components/hab/src/command/cli/setup.rs

```diff
@@ -110,20 +110,19 @@ pub fn start(ui: &mut UI, cache_path: &Path, analytics_path: &Path) -> Result<()
     } else {
         ui.para("Okay, maybe another time.")?;
     }
-    ui.heading("GitHub Access Token")?;
+    ui.heading("Habitat Personal Access Token")?;
     ui.para(
         "While you can build and run Habitat packages without sharing them on the public \
                depot, doing so allows you to collaborate with the Habitat community. In \
                addition, it is how you can perform continuous deployment with Habitat.",
     )?;
     ui.para(
-        "The GitHub personal access token needs the user:email and read:org OAuth scopes. \
-               Habitat uses the information provided through these scopes for \
-               authentication and to determine features based on team membership. You can \
-               set this up at https://github.com/settings/tokens",
+        "The Habitat personal access token can be generated via the Habitat Builder web \
+               site Profile page (https://bldr.habitat.sh/#/profile). Once you have \
+               generated your token, you can enter it here.",
     )?;
     ui.para(
-        "If you would like to share your packages on the depot, please enter your GitHub \
+        "If you would like to share your packages on the depot, please enter your Habitat \
```

Comment:
Are we keeping the term "depot" as a user-facing term? Sometime we say "Habitat Builder web site" and other times we say "depot". I found it a little confusing when I started, and I got the sense that maybe we were moving away from it in UI text. Most of the references are right here it seems, so this might be a good time to update them.

Reply:
I went through a lot of docs during the launch and corrected depot->builder. We should be calling it Builder in the public for sure. Depot is just a component of Builder